test/test.js

```diff
@@ -83,16 +94,143 @@ describe( "svg-shield", function ()
+        {
+            assert.equal( err, null );
+            assert.equal( res,
+            '<svg xmlns="http://www.w3.org/2000/svg" width="150" height="20">\n\
+    <defs>\n\
+        <filter id="drop-shadow">\n\
+            <feOffset result="offOut" in="SourceAlpha" dx="0.5" dy="1" />\n\
+            <feGaussianBlur result="blurOut" in="offOut" stdDeviation="0.5" />\n\
+            <feBlend in="SourceGraphic" in2="blurOut" mode="normal" />\n\
+        </filter>\n\
+    </defs>\n\
+    <mask id="round-rect"><rect width="150" height="20" rx="5" ry="5" fill="#fff"/></mask>\n\
+    <g mask="url(#round-rect)">\n\
+        <path fill="#333" d="M0 0h100v20H0z"/>\n\
+        <path fill="#000" d="M100 0h60v20H100z"/>\n\
+    </g>\n\
+    <g fill="#fff" text-anchor="middle" font-family="Verdana, sans-serif" font-size="8">\n\
+        <text x="50" y="14" filter="url(#drop-shadow)">name</text>\n\
+        <text x="125" y="14" filter="url(#drop-shadow)">value</text>\n\
+    </g>\n\
+</svg>' );
+
+            done();
+        } );
+    } );
+
+    it( "should get a shield with min 0 corner radius", function ( done )
+    {
+        var options = {
+            valueWidth: 50,
+            nameWidth: 100,
+            valueBgColor: "#000",
+            nameBgColor: "#333",
+            name: "name",
+            value: "value",
+            fontFamily: "Verdana, sans-serif",
+            fontSize: 12,
+            radius: -6
+        };
+
+        shield.getShield( options, function ( err, res )
+        {
+            assert.equal( err, null );
+            assert.equal( res,
+            '<svg xmlns="http://www.w3.org/2000/svg" width="150" height="20">\n\
+    <defs>\n\
+        <filter id="drop-shadow">\n\
+            <feOffset result="offOut" in="SourceAlpha" dx="0.5" dy="1" />\n\
+            <feGaussianBlur result="blurOut" in="offOut" stdDeviation="0.5" />\n\
+            <feBlend in="SourceGraphic" in2="blurOut" mode="normal" />\n\
+        </filter>\n\
+    </defs>\n\
+    <mask id="round-rect"><rect width="150" height="20" rx="0" ry="0" fill="#fff"/></mask>\n\
+    <g mask="url(#round-rect)">\n\
+        <path fill="#333" d="M0 0h100v20H0z"/>\n\
+        <path fill="#000" d="M100 0h60v20H100z"/>\n\
+    </g>\n\
+    <g fill="#fff" text-anchor="middle" font-family="Verdana, sans-serif" font-size="12">\n\
+        <text x="50" y="14" filter="url(#drop-shadow)">name</text>\n\
+        <text x="125" y="14" filter="url(#drop-shadow)">value</text>\n\
+    </g>\n\
+</svg>' );
+
+            done();
+        } );
+    } );
+
+    it( "should get a shield with max 10 corner radius", function ( done )
+    {
+        var options = {
+            valueWidth: 50,
+            nameWidth: 100,
+            valueBgColor: "#000",
+            nameBgColor: "#333",
+            name: "name",
+            value: "value",
+            fontFamily: "Verdana, sans-serif",
+            fontSize: 12,
+            radius: 23
+        };
```

Comment:
For these 3 tests, I feel like given the name of the test `should get a shield with max 10 corner radius` it would make more sense to just use the default options except for `radius` in this case, or the relevant items above.